Central and Eastern European Online Library Journals.js

```diff
@@ -0,0 +1,308 @@
+{
+	"translatorID": "19cef926-c5b6-42e2-a91c-6f2722f8b36d",
+	"label": "Central and Eastern European Online Library Journals",
+	"creator": "Timotheus Kim",
+	"target": "^https://www\\.ceeol\\.com/search",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2021-06-25 15:07:56"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2021 Timotheus Kim
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes('article')) return "journalArticle";
```

Comment:
Could the URL fragment you're matching on just be `/article-detail?`? `article` alone will yield a lot of false positives.

Reply:
Thanks for code reviewing and your suggestion. I have changed it.